### PR TITLE
New Playbook: dispatch_playbooks

### PR DESCRIPTION
**Pull request by kelby-shelton**
# New playbook called dispatch_playbooks

## Description
An alternative to the built-in playbook launcher that allows a user to launch multiple playbooks in one block based on dynamic data.

## Features
- Collects indicator data from event and routes it to specified input style playbooks.
- Attempting to launch community input playbooks will cause error.
- Waits for child playbooks to finish and collects their outputs and passes them forward.

### CODE REVIEW

File: dispatch_playbooks.py [hunk 1]
```diff
@@ -0,0 +1,406 @@
...
+        indicator_tags_exclude.append(indicator_set[1])
+            
+    # build parameters list for as a connected group
+    parameters.append({
+        "container": id_value,
+        "indicator_types_include": ', '.join([item for item in find_supported_indicator_types__list if item]),
+        "indicator_types_exclude": None,
+        "indicator_tags_include": ', '.join([item[0] for item in indicator_tags_include if item]),
+        "indicator_tags_exclude": ', '.join([item[0] for item in indicator_tags_exclude if item]),
+        "artifact_ids_include": playbook_input_artifact_ids_include_values,
```
Comment (philroyer-phantom) [inline dispatch_playbooks.py]: I am seeing an exception in the CF here if we pass in [""] when this playbook is called with no artifact IDs. How about changing this line to the following:
`        "artifact_ids_include": ', '.join([item[0] for item in playbook_input_artifact_ids_include_values if item]),`
?

**Review by philroyer-phantom — COMMENTED** [review]
See line 335.

**Comment by philroyer-phantom** [comment]
Looks like there are two files left over from before the rename. Could you remove dispatch_playbooks.json and dispatch_playbooks.png please?

File: dispatch_input_playbooks.json [hunk 2]
```diff
@@ -0,0 +1,614 @@
...
+                        "type": "decision"
+                    },
+                    "errors": {},
+                    "id": "3",
+                    "type": "decision",
+                    "x": 80,
+                    "y": 474
+                }
+            },
+            "notes": "- Must have at least one input style playbook in whichever repo you choose for this utility to work.\n- Playbook will error out if community repo is provided to ensure that users do not accidentally launch community input playbooks that have not been configured."
```
Comment (philroyer-phantom) [inline dispatch_input_playbooks.json]: Please add a note that artifact scope will be ignored. That can be a bit surprising for someone not used to how the "indicator_collect" function works.

Comment (kelby-shelton) [reply dispatch_input_playbooks.json]: Done

**Pull request merged**